budgetportal/tests/test_provincial_infrastructure_projects.py

```diff
@@ -17,8 +21,10 @@
 EMAIL = "testuser@domain.com"
 PASSWORD = "12345"
 
+random.seed(123456789)
```

Comment:
remove now it's unused

Reply:
removed

https://github.com/vulekamali/datamanager/pull/190/commits/21ead8374fd5b19678775974b76ca83037aad7ba